CONTRIBUTING.md

```diff
@@ -0,0 +1,74 @@
+# ![TTTReborn logo](https://github.com/TTTReborn/graphics/raw/main/github_social/social.png) Contributing to TTTReborn
```

Comment:
looking at the rich diff, the banner might be a bit too much. Thoughts?

Reply:
You can use html markup to reduce the size and even center it etc.

Also you might want to change the image to another one that is only the logo